src/Day.js

```diff
@@ -150,7 +149,8 @@ export default class Day extends Component {
         onTouchStart={handleEvent(onTouchStart, day, modifiers)}
         onFocus={handleEvent(onFocus, day, modifiers)}
       >
-        {children}
+        <span className="DayPicker-ScreenReaderOnly">{ariaLabel}</span>
```

Comment:
User can set their own [classNames](http://react-day-picker.js.org/api/DayPicker#classNames): this className should be added to the list.

Reply:
this className is always required to hide this from a typical user's view. are you asking that a key on the `classNames` object be added for the screenreader text?

Reply:
Yes! Every class name used in the component can be replaced with a custom one via that `classNames` prop. We need to add it here https://github.com/gpbl/react-day-picker/blob/v7/src/classNames.js.